react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -342,9 +347,14 @@ export default class Map3DLayer extends CompositeLayer<
 > {
     initializeState(): void {
         // Load mesh and texture and store in state.
+        const isBounds = typeof this.props.bounds !== "undefined";
+        const bounds = (
```

Comment:
This should be the other way around; if `frame` is defined, then it should be used instead of `bounds`.

Also, would be useful with a deprecation warning here if `bounds` are specified. Eg.

```
console.warn("\"bounds\" is deprecated. Use \"frame\" instead.);
```